google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineHelper.java

```diff
@@ -43,16 +45,16 @@
   /**
    * The default app.yaml to use.
    */
-  Path defaultAppYaml();
+  Optional<Path> defaultAppYaml();
 
   /**
    * The default Dockerfile we suggest for custom flexible deployments.
    *
    * @param deploymentArtifactType depending on the artifact type we provide a different default
    *                               Dockerfile
-   * @return A {@link java.io.File} path to the default Dockerfile
+   * @return An {@link Optional} containing a {@link Path} to the default Dockerfile
```

Comment:
Good catch. Sounds like some format checker tools aren't configured properly, or I'm not using them properly.